packages/mobile/src/app/App.tsx

```diff
@@ -70,7 +70,11 @@ export class App extends React.Component<Props> {
     // Handles opening Clevertap deeplinks when app is closed / in background
     CleverTap.getInitialUrl(async (err: any, url) => {
```

Comment:
if the callback were taken out and exposed (with something like `export const cleverTapGetInitialUrlCallback = ...`) then it could be tested pretty easily